x-pack/test/api_integration/apis/transform/delete_transforms.ts

```diff
@@ -73,7 +79,6 @@ export default ({ getService }: FtrProviderContext) => {
         expect(body[transformId].transformJobDeleted.success).to.eql(true);
         expect(body[transformId].destIndexDeleted.success).to.eql(false);
         expect(body[transformId].destIndexPatternDeleted.success).to.eql(false);
-        await transform.api.waitForIndicesToExist(destinationIndex);
```

Comment:
Why did you remove that check? I think it's still good to know that the dest index wasn't deleted.